# HG changeset patch
# User Guillem Barba <guillembarba@gmail.com>
remove digits attribute from internal quantity fields

issue4741
review19281002

Index: move.py
===================================================================

--- a/trytond/trytond/modules/product_cost_fifo/move.py
+++ b/trytond/trytond/modules/product_cost_fifo/move.py
@@ -11,9 +11,7 @@
 
 class Move:
     __name__ = 'stock.move'
-    fifo_quantity = fields.Float('FIFO Quantity',
-        digits=(16, Eval('unit_digits', 2)),
-        depends=['unit_digits'])
+    fifo_quantity = fields.Float('FIFO Quantity')
 
     @classmethod
     def __setup__(cls):

